packages/client/src/context/Auth.context.tsx

```diff
@@ -52,12 +59,15 @@ export const AuthProvider: FC<AuthProviderProps> = ({ children }) => {
   const [authenticated, setAuthenticated] = useState<boolean>(true);
   const [decodedToken, setDecodedToken] = useState<DecodedToken | null>(null);
   const [organization, setOrganization] = useState<Organization | null>(null);
+  const [organizationList, setOrganizationList] = useState<Organization[] | null>(null);
```

Comment:
I'd recommend having the type be

```
useState<Organization[]>([])
```

That way we don't have to have null checks other places and having an empty list is valid when the data hasn't loaded in yet.